imports/api/cases.js

```diff
@@ -106,44 +106,43 @@ if (Meteor.isServer) {
     withUsers(caseItem => _.flatten(Object.values(getCaseUsers(caseItem))).map(u => u.login))
   ))
   // TODO: Add tests for this
-  Meteor.publish(`${collectionName}.associatedWithMe`, associationFactory(
-    publicationObj.publishByCustomQuery({
-      uriTemplate: () => '/rest/bug',
-      queryBuilder: subHandle => {
-        if (!subHandle.userId) {
-          return {}
-        }
-        const currUser = Meteor.users.findOne(subHandle.userId)
-        const { login: userIdentifier } = currUser.bugzillaCreds
-        return {
-          f1: 'keywords',
-          o1: 'nowords',
-          v1: 'inspection_report',
-          f2: 'OP',
-          j2: 'OR',
-          f3: 'assigned_to',
-          o3: 'equals',
-          v3: userIdentifier,
-          f4: 'cc',
-          o4: 'substring',
-          v4: userIdentifier,
-          f5: 'reporter',
-          o5: 'equals',
-          v5: userIdentifier,
-          list_id: '78',
-          query_format: 'advanced',
-          include_fields: 'product,summary,id,status,assigned_to'
-        }
-      },
-      addedMatcherFactory: strQuery => {
-        const { v1: userIdentifier } = JSON.parse(strQuery)
-        return caseItem => {
-          const { assignee, creator, involvedList, keywords } = transformCaseForClient(caseItem)
-          return (
-            !(keywords && keywords.includes('inspection_report')) && (
-              userIdentifier === assignee ||
-              userIdentifier === creator ||
-              involvedList.includes(userIdentifier)
+  Meteor.publish(`${collectionName}.associatedWithMe`, publicationObj.publishByCustomQuery({
+    uriTemplate: () => '/rest/bug',
+    queryBuilder: subHandle => {
+      if (!subHandle.userId) {
+        return {}
+      }
+      const currUser = Meteor.users.findOne(subHandle.userId)
+      const { login: userIdentifier } = currUser.bugzillaCreds
+      return {
+        f1: 'keywords',
+        o1: 'nowords',
+        v1: 'inspection_report',
+        f2: 'OP',
+        j2: 'OR',
+        f3: 'assigned_to',
+        o3: 'equals',
+        v3: userIdentifier,
+        f4: 'cc',
+        o4: 'substring',
+        v4: userIdentifier,
+        f5: 'reporter',
+        o5: 'equals',
+        v5: userIdentifier,
+        list_id: '78',
+        query_format: 'advanced',
+        include_fields: 'product,summary,id,status,assigned_to'
+      }
+    },
+    addedMatcherFactory: strQuery => {
+      const { v1: userIdentifier } = JSON.parse(strQuery)
+      return caseItem => {
+        const { assignee, creator, involvedList, keywords } = transformCaseForClient(caseItem)
+        return (
+          !(keywords && keywords.includes('inspection_report')) && (
+            userIdentifier === assignee ||
+            userIdentifier === creator ||
+            involvedList.includes(userIdentifier)
```

Comment:
It looks like you've overridden newer code with the previous code during merge. I think it would be broken now